### PR TITLE
Reformat and minor Markdown touch-ups (attempt 2)

### PR DESCRIPTION
**Pull request by Chinay-Domitrix**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,69 +1,73 @@
...
+Where this overview gets really exciting is when we dive into [macros](#macros). If you're here to give Passerine a
+try, [skip to Installation](#installation).
+
+> **:warning:Note that Passerine is a _work in progress_: features mentioned in this overview may not be implemented yet.**
 
 ### Syntax
-The goal of Passerine's syntax is to make all expressions as *concise* as possible while still conserving the 'feel' of *different types* of expressions.
+
+The goal of Passerine's syntax is to make all expressions as _concise_ as possible while still conserving the 'feel'
+of _different types_ of expressions.
```
Comment (slightknack) [inline README.md:70]: Two quick things, then I'll merge this PR:

1. I think your formatter added 80-column line breaks to the markdown file. Please remove them
2. The italics `*hello*` are still being changed to `_hello_`.

File: README.md [hunk 2]
```diff
@@ -72,96 +76,96 @@ square = x -> x * x
...
 
 #### A Quick(-sort) Example
+
 Here's another slightly more complex example – a recursive quick-sort with questionable pivot selection:
 
 ```elm
 sort = list -> match list {
     -- base case
     [] -> []
-
```
Comment (slightknack) [inline README.md:149]: These newlines are here on purpose, same for all other listings.

File: README.md [hunk 3]
```diff
@@ -249,24 +265,15 @@ In the last section, we touched on pattern matching a little. I hope to now go o
...
...------------------------------------- | | variable | `foo` | Terminal pattern, binds a variable to a value. | | data | `420.69` | Terminal pattern, data that _must_ match, raises an error otherwise. See the following section on fibers and concurrency to get an idea of how errors work in Passerine. | | discard | `_` | Terminal pattern, matches any data, does not produce a binding. | | label | `Baz p` | Matches a label, i.e. a named _type_ in Passerine. | | tuple | `(p₀, ...)` | Matches each element of a tuple, which is a group of elements, of potentially different types. Unit `()` is the empty tuple. | | list | `[]`/`[p₀, ..p₁]` | `[]` Matches an empty list - `p₀` matches the head of a list, `..p₁` matches the tail. | | record | `{f₀: p₀, ...}` | A record, i.e. a struct. This is a series of field-pattern pairs. If a field does not exist in the target record, an error is raised. | | enum | `{p₀; ...}` | An enumeration, i.e. a union. Matches if any of the patterns hold. | | is | `p₀: p₁` | A type annotation. Matches against `p₀` only if `p₁` holds, errors otherwise. | | where | `p \| e` | A bit different from the other patterns so far. Matches `p` only if the expression `e` is true. |
```
Comment (slightknack) [inline README.md:276]: I think this table got botched...

**Review by slightknack — CHANGES_REQUESTED** [review]
Thanks for incorporating the feedback on the last PR! There are just a couple of things, and then it'll be ready for merging.

**Comment by Chinay-Domitrix** [comment]
I'll take a look and fix everything in a bit

**Comment by slightknack** [comment]
Any updates?

**Comment by slightknack** [comment]
With the introduction of the (WIP) [Codex](https://codex.passerine.io), a lot of information in the README will be moving there. Because of this, the README will look a lot different than it does today (especially once #52 is merged). Because of this, I'm closing this PR. If you'd like to help write the Codex, feel free to open a PR [on its repo](https://github.com/vrtbl/codex).

**Pull request closed without merging**